wgpu-types/src/lib.rs

```diff
@@ -1680,6 +1696,14 @@ pub enum VertexFormat {
     Int3 = 28,
     /// Four signed ints (i32). `ivec4` in shaders.
     Int4 = 29,
+    /// One double-precision float (f64). `double` in shaders. Requires SHADER_FLOAT64 and VERTEX_ATTRIBUTE_64BIT features.
```

Comment:
similarly, let's adjust the comment to only mention `VERTEX_ATTRIBUTE_64BIT` here